code/modules/uplink/uplink_items.dm

```diff
@@ -545,11 +545,10 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "A short bow mounted across a tiller in miniature. \
 	Small enough to fit into a pocket or slip into a bag unnoticed. \
 	It will synthesize and fire bolts tipped with a debilitating \
-	toxin that will damage and disorient targets, causing them to \
-	slur as if inebriated. It can produce an infinite number \
-	of bolts, but takes time to automatically recharge after each shot."
-	item = /obj/item/gun/energy/kinetic_accelerator/crossbow
-	cost = 10
+	toxin that will damage and weaken targets; It can store up to ten \
+	bolts, and recharges them over time."
+	item = /obj/item/gun/energy/crossbow
+	cost = 11
```

Comment:
Undo this change, on the next line insert:
`limited_stock = 1`

Reply:
But pkpenguiin I also like the price being 11tc regardless of whether or not you can buy two

Reply:
I do not

Reply:
ebow has always been good as fuck mate I doubt this'll make it anything but less obvious a choice

Reply:
RIP ebow i guess